src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -29,8 +29,8 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "java_bytecode_parser.h"
 #include "java_class_loader.h"
 #include "java_utils.h"
-#include <java_bytecode/ci_lazy_methods.h>
-#include <java_bytecode/generate_java_generic_type.h>
+#include "ci_lazy_methods.h"
+#include "java_bytecode/generate_java_generic_type.h"
```

Comment:
What is the purpose of this change? If intentional it should be consistent - since both files are in `java_bytecode` they either should both prefix it (which I think is what the coding guidelines say) or neither (which seems more consistent with the other includes in this file)

Reply:
I'd drop this as unrelated cleanup

Reply:
I got rid of this (reset it to what it was), sorry for the confusion. I originally added more includes here, tried to change the existing ones to make them more consistent, then ended up in a half-finished state and forgot to change them back.